src/tictactoe/marker_validation.clj

```diff
@@ -0,0 +1,33 @@
+(ns tictactoe.marker-validation
+	(:require [tictactoe.user-interface :as ui]
```

Comment:
Formatting? I'm going to start cracking down on wat formatting. When you put your PRs up just glance over them and look for formatting issues. Ideally, when you join a team, you'll have a tool which lints the code if not auto format it for you.